doc/release-notes.md

```diff
@@ -63,6 +63,8 @@ P2P and network changes
 
 Updated RPCs
 ------------
+- `getmempoolentry` and `getrawmempool true` no longer return top level fee fields `fee`, `modifiedfee`, `ancestorfees`, `descendantfees`. These fields were deprecated in `0.17` and moved to the `fees` sub-object in the response.
+The `-deprecated=fees` flag must be passed for these fields to be included in the top level response. Note: this flag will only be available until `v24`, after which the fields will be fully removed. It is recommended you migrate to using the `fees` sub-object in the response
```

Comment:
suggestions

- add the PR number
- add  `getmempoolancestors` and `getmempooldescendants` that are also changed
- s/true/verbose=true/ for clarity
- wrap the lines
- s/sub-object/JSON object/
- s/top level/top-level/
- a few others

```diff
-- `getmempoolentry` and `getrawmempool true` no longer return top level fee fields `fee`, `modifiedfee`, `ancestorfees`, `descendantfees`. These fields were deprecated in `0.17` and moved to the `fees` sub-object in the response.
-The `-deprecated=fees` flag must be passed for these fields to be included in the top level response. Note: this flag will only be available until `v24`, after which the fields will be fully removed. It is recommended you migrate to using the `fees` sub-object in the response
+- RPCs `getmempoolentry`, `getrawmempool verbose=true`, `getmempoolancestors
+  verbose=true` and `getmempooldescendants verbose=true` no longer return the
+  top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and
+  `descendantfees`. These fields were deprecated in v0.17 and moved to the
+  `fees` JSON object in the response. The `-deprecated=fees` configuration
+  option must be passed for these fields to be included in the top-level
+  response. This option will only be available until v24.0, after which the
+  fields will be fully removed. Use the `fees` JSON object for these RPCS
+  instead. (#22689)
```